person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```

Comment:
Nice job implementing the of_age? method👍🏼. just as required in this milestone, the `of_age?` method is meant to be a private method😉. currently, you have implemented it as a public method. Kindly implement it as a private method👍🏼.
<img width="818" alt="image" src="https://github.com/tan12082001/Project-OOP-School-Library/assets/64643995/440d4d18-e92c-4f77-97f6-a7247ed20de7">


```suggestion

  def can_use_services?
    of_age? || @parent_permission
  end
  
  private

  def of_age?
    @age >= 18
  end
```